react/components/BaseProductIdentifier.tsx

```diff
@@ -27,25 +28,32 @@ const messages = defineMessages({
   },
 })
 
+const CSS_HANDLES = [
+  'product-identifier',
+  'product-identifierLabel',
+  'product-identifierSeparator',
+  'product-identifierValue'
+]
+
 const ProductIdentifier: StorefrontFunctionComponent<
   ProductIdentifierProps & InjectedIntlProps
 > = ({ idField, value, customLabel, label, intl }) => {
   return (
     <span
-      className={`${styles['product-identifier']} ${styles['product-identifier']}--${idField} c-muted-1`}
+      className={`${handles.product - identifier} ${handles.product - identifier}--${idField} c-muted-1`}
     >
       {label !== 'hide' && (
-        <span className={styles['product-identifier__label']}>
+        <span className={handles.product - identifierLabel}>
           {label === 'custom' && <IOMessage id={customLabel} intl={intl} />}
           {label === 'default' && (
             <FormattedMessage id={messages[idField].id} />
           )}
         </span>
       )}
       {label !== 'hide' && (
-        <span className={styles['product-identifier__separator']}>: </span>
+        <span className={handles.product - identifierSeparator}>: </span>
       )}
-      <span className={styles['product-identifier__value']}>{value}</span>
+      <span className={handles.product - identifierValue}>{value}</span>
```

Comment:
The `handles` variable does not exist in this scope.
You should declare it inside of the function component like:

`const handles = useCssHandles(CSS_HANDLES)`